repository/result.c

```diff
@@ -1,33 +1,33 @@
 #include <mysql/mysql.h>
+#include <stdbool.h>
```

Comment:
if you add this to the header file, you shouldn't need to add it the source file as well